### DISCUSSION TITLE
Import from 'Rezeptbuch 3.0'

### DISCUSSION THREAD

**Discussion by tuetenk0pp** [TandoorRecipes/recipes#375] [Q&A] [answered]
I wonder if anyone has tried to import recipes originating from the 'Rezeptbuch' PC-Software. Rezeptbuch itself can export to the RezKonv format which is apparently quite popular. Furthermore MealMaster, and Cookmate (MCB-format).

**Comment by vabene1111** [comment, marked as answer]
This is most likely not possible but i am working on supporting more import methods. 

Can you open an issue with with an example export of the format so i can use it for testing once i get around to implenting it ?

Please creaate one issue per type, i think MealMaster already has one.

**Reply by tuetenk0pp** [reply]
Ok I just wrote the developer of Rezeptbuch an email sking for details about the format. I will supply them in the issue
